src/syntaxtree.c

```diff
@@ -107,16 +128,22 @@ static Obj SyntaxTreeArgcompInt(UInt i)
     return INTOBJ_INT(i);
 }
 
+static Expr SyntaxTreeArgCodeInt(Obj i)
+{
+    return INT_INTOBJ(i);
+}
```

Comment:
This is potentially unsafe, and performs no input validation. I suggest that we simply replace `SyntaxTreeArgCodeInt` by `UInt_ObjInt` everywhere. And `SyntaxTreeArgcompInt` should be replaced by `ObjInt_UInt`. I didn't make the latter change in my branch yesterday to reduce the amount of rebasing work for you ;-)

Reply:
Done.